### ISSUE TITLE
Compilation error with `--enable-debug` on master

### ISSUE THREAD

**Issue by rohgarg** [dmtcp/dmtcp#332]
util_init.cpp fails to compile if I configure DMTCP with `--enable-debug`. Here's the error message:

``` bash
make[2]: Entering directory '/home/rohgarg/proj/rohgarg-github/dmtcp-4/dmtcp/src'
g++ -DHAVE_CONFIG_H -I. -I../include  -I../include -I../jalib  -fPIC -DDEBUG -g -O0 -Wall -MT util_init.o -MD -MP -MF .deps/util_init.Tpo -c -o util_init.o util_init.cpp
util_init.cpp: In function ‘void dmtcp::Util::initializeLogFile(dmtcp::string, dmtcp::string, dmtcp::string)’:
util_init.cpp:154:23: error: ‘JLOG’ was not declared in this scope
   JLOG(a.str().c_str());
```

**Comment by rohgarg** [comment]
The bug was introduced in commit: d46b392. In the commit the `JLOG()` macro was deleted but all of its references were not cleaned up.

**Comment by rohgarg** [comment]
Fixed via PR #460.
